global_planner/src/nodes/global_planner_node.cpp

```diff
@@ -103,6 +103,9 @@ void GlobalPlannerNode::readParams() {
   nh_.param<double>("start_pos_z", start_pos_.z, 3.5);
   global_planner_.goal_pos_ =
       GoalCell(start_pos_.x, start_pos_.y, start_pos_.z);
+  double robot_radius;
+  nh_.param<double>("robot_radius", robot_radius, 0.5);
```

Comment:
```suggestion
  nh_.param<double>("robot_radius", robot_radius, 1.0);
```
For the consistency with global_planner_octomap.launch file